jitbuilder/control/Jit.cpp

```diff
@@ -189,19 +189,7 @@ internal_compileMethodBuilder(TR::MethodBuilder *m, void **entry)
    {
    auto rc = m->Compile(entry);
 
-#if defined(J9ZOS390)
-   struct FunctionDescriptor
-   {
-      uint64_t environment;
-      void* func;
-   };
-
-   FunctionDescriptor* fd = new FunctionDescriptor();
-   fd->environment = 0;
-   fd->func = *entry;
-
-   *entry = (void*) fd;
-#elif defined(AIXPPC)
```

Comment:
Finally! This hack is now removed and we don't need to do anything special for z/OS. Once AIX gets support we can also remove the AIX piece as well.

Reply:
Sweet!